documentation/source/user_section/tutorials/registration-to-template/lumbar-registration/template-registration.rst

```diff
@@ -0,0 +1,36 @@
+.. _applying-lumbar-registration-algorithm:
+
+Applying the registration algorithm
+###################################
+
+To apply the registration algorithm, the following command is used:
+
+.. code:: sh
+
+   sct_register_to_template -i t2_crop.nii.gz \
```

Comment:
To I need to go to a specific branch to get the updated PAM50 with the 60 label?

Reply:
Fixed by https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4250/commits/1dabb41e6cba1dfda42b18b6a6a95e7ab5fcc423.